lib/nuntiux.ex

```diff
@@ -11,14 +11,18 @@ defmodule Nuntiux do
   @type history :: Nuntiux.Mocker.history()
   @type received? :: Nuntiux.Mocker.received?()
   @type event :: Nuntiux.Mocker.event()
+  @type expect_fun :: Nuntiux.Mocker.expect_fun()
+  @type expect_name :: Nuntiux.Mocker.expect_name()
+  @type expect_id :: Nuntiux.Mocker.expect_id()
+  @type expects :: Nuntiux.Mocker.expects()
 
   defmacro if_mocked(process_name, fun) do
     quote bind_quoted: [
             process_name: process_name,
             fun: fun
           ] do
       if process_name in mocked(),
-        do: fun.(process_name),
+        do: fun.(),
```

Comment:
Since all the functions that use this macro also have `process_name` in their scope, this is not only redundant but adds to confusion for no specific reason.